tests/transformers/test_transformers_model_export.py

```diff
@@ -1341,6 +1363,46 @@ def test_qa_pipeline_pyfunc_load_and_infer(small_qa_pipeline, model_path, infere
     assert all(isinstance(element, str) for element in inference)
 
 
+def read_image(filename):
+    image_path = os.path.join(pathlib.Path(__file__).parent.parent, "datasets", filename)
```

Comment:
Why are we mixing pathlib.Path and os.path.join() ? Just use pathlib to traverse the paths and read the bytes of the file directly.

Reply:
fixed